corinaorasan/SleepTracker
# Add a recycler view

## Description
Replace main ScrollView with RecyclerView, then implement an Adapter for displaying list items.
### [Task]

https://classroom.udacity.com/courses/ud9012/lessons/ee5a525f-0ba3-4d25-ba29-1fa1d6c567b8/concepts/8d0a8eb2-251e-4682-a105-da17fbec8ccd

## Steps to Test or Reproduce
1.Start the app
2.Press START 
3.Sleep
4.Press STOP when you wake up
5.Press click on the smiley face that represents your sleep quality 
6.Observe the sleep quality numbers list
